sympy/printing/latex.py

```diff
@@ -205,9 +205,9 @@ def __init__(self, settings=None):
             self._settings['imaginary_unit_latex'] = \
                 self._settings['imaginary_unit']
 
-    def parenthesize(self, item, level, strict=False):
+    def parenthesize(self, item, level, is_neg=False, strict=False):
         prec_val = precedence_traditional(item)
-        if (prec_val < level) or ((not strict) and prec_val <= level):
+        if (((prec_val < level) or ((not strict) and prec_val <= level)) or (is_neg and strict)):
```

Comment:
Is the outer parenthesis necessary?

Reply:
I added it for clearity as there are a lot of checks in single if statement.

Reply:
What about pulling the logic outside, and rather make copy-paste, if you want to make it clear that a lot of checks are done in parallel?
```python3
if is_neg and strict:
    return r"\left({}\right)".format(self._print(item))
if (prec_val < level) or ((not strict) and prec_val <= level):
    return r"\left({}\right)".format(self._print(item))
```

Reply:
Actually i was thinking about this.
I will commit with these changes.